spec/formatting.md

```diff
@@ -2,13 +2,16 @@
 
 ## Introduction
 
-This document defines the behaviour of a MessageFormat 2.0 implementation
+This document defines the behavior of a MessageFormat 2.0 implementation
 when formatting a message for display in a user interface, or for some later processing.
 
-To start, we presume that a _message_ has either been parsed from its syntax
-or created from a data model description.
-If this construction has encountered any _Syntax Errors_ or _Data Model Errors_,
-an appropriate error MUST be emitted and a _fallback value_ MAY be used as the formatting result.
+To start, parse the _message_ from its syntax or create it from a data model description.
+
+If the message being formatted has any _Syntax Errors_ or _Data Model Errors_,
+an appropriate error MUST be emitted and
+the result of formatting the _message_ MUST be a pattern resolving to a single _fallback value_
```

Comment:
This implies that no syntax errors are recoverable, right? For example, that:

```
.local $bar {|foo|} {{{$bar}}}
```

formats to `�` and not `{$bar}`.

It would be an improvement to disambiguate the spec in that way, but I'm not sure if that's the behavior everyone wants?

Reply:
I'm having a hard time figuring out a situation where it would be overall beneficial to allow for messages with syntax errors to format to anything other than `{�}` or something like `{msg-id}`, in particular if that behaviour is different between implementations.

What is the process generating such a broken message that we want to support?